tests/e2e/specs/user-input-questions.test.js

```diff
@@ -36,66 +36,52 @@ import {
 	setSearchConsoleProperty,
 } from '../utils';
 
-const wpConditionalDescribe = () => {
```

Comment:
This was no longer relevant as we don't run E2E against 4.x versions of WP anymore.